Cql/CqlToElmTests/NotTest.cs

```diff
@@ -153,6 +153,6 @@ public void Not_Null()
                 library NotTest version '1.0.0'
 
                 define private Not_1: not 1
-            ", "Cannot resolve call*first argument*Integer*cannot implicitly be cast*Boolean.");
+            ", "Could not resolve*");
```

Comment:
Yes, I tried to match the exact messages that the cql-to-elm reference implementation does.  The messages are mostly the same, but it's phrased "Could not" instead of "cannot".